### PR TITLE
feat: add method for decoding sealed results

### PR DESCRIPTION
**Pull request by TheUnderScorer**

### CODE REVIEW

File: src/FingerprintPro.ServerSdk/Sealed.cs [hunk 1]
```diff
@@ -0,0 +1,154 @@
+using System.IO.Compression;
```
Comment (ilfa) [inline src/FingerprintPro.ServerSdk/Sealed.cs:1]: Is it correct in terms of .NET best practice to keep this file in the root?

**Review by ilfa — CHANGES_REQUESTED** [review]

File: src/FingerprintPro.ServerSdk/Sealed.cs (hunk 1 above)
Comment (TheUnderScorer) [reply src/FingerprintPro.ServerSdk/Sealed.cs:1]: The directories must match the namespace, but I think it's okay to keep this class in root.

File: README.md [hunk 2]
```diff
@@ -91,6 +91,24 @@ var configuration = new Configuration(Environment.GetEnvironmentVariable("API_KE
...
+```csharp
+using FingerprintPro.ServerSdk;
+
+var sealedResult = Environment.GetEnvironmentVariable("BASE64_SEALED_RESULT")!;
+var sealedKey = Environment.GetEnvironmentVariable("BASE64_KEY")!;
+
+var events = Sealed.UnsealEventResponse(Convert.FromBase64String(sealedResult), new[]
+{
+new Sealed.DecryptionKey(Convert.FromBase64String(sealedKey), Sealed.DecryptionAlgorithm.Aes256Gcm)
+});
```
[the remark this answers is not in the document]
Comment (TheUnderScorer) [reply README.md:106]: Done!

**Review by ilfa — APPROVED** [review]

**Pull request merged**